### ISSUE TITLE
[firehol_level3]: stackpath CDN blocked

### ISSUE THREAD

**Issue by jpmorrison** [firehol/blocklist-ipsets#123]
151.139.128.10 is blocked affecting multiple sites

Reported in https://github.com/firehol/blocklist-ipsets/issues/116 but not fixed

https://www.stackpath.com/
https://kit.fontawesome.com

static.streameye.net.   2644    IN      CNAME   h9j6p4t8.stackpathcdn.com.
h9j6p4t8.stackpathcdn.com. 1267 IN      A       151.139.128.10

**Comment by aaww01** [comment]
Yup, reported to Font-Awesome in https://github.com/FortAwesome/Font-Awesome/issues/15812 
I suppose they could change CDN as the ultimate fix.

**Comment by bmoultonsrq** [comment]
hi
